lib/config/guiconfiguration.go

```diff
@@ -119,7 +119,13 @@ var bcryptExpr = regexp.MustCompile(`^\$2[aby]\$\d+\$.{50,}`)
 // SetPassword takes a bcrypt hash or a plaintext password and stores it.
 // Plaintext passwords are hashed. Returns an error if the password is not
 // valid.
+// If the plaintext password is empty, the password is unset instead.
 func (c *GUIConfiguration) SetPassword(password string) error {
+	if password == "" {
+		c.Password = ""
+		return nil
+	}
+
```

Comment:
Yeah. I added this way back in #8417, I don't remember exactly why but probably because I personally will want to disable password auth and use only WebAuthn when this (hopefully) lands.